.ci/on_conan_release.jenkinsfile

```diff
@@ -0,0 +1,87 @@
+/*
+
+    Triggered whenever a new Conan version is uploaded to pypi.
+
+*/
+
+node('Linux') {
+    stage('Input parameters') {
+        echo """
+        - conan_version: ${params.conan_version}
+        - build_new_images: ${params.build_new_images}
```

Comment:
Could you comment in this code, how these 3 variables are filled?

Reply:
These are parameters/inputs to the Jenkins job. We are going to trigger it Jenkins shows the appropriate form: 
![image](https://user-images.githubusercontent.com/1406456/147738820-31c7045c-a29e-4617-bc21-a3c8c81a36ee.png)

If called from other jobs, they just need to provide the values.